bundles/org.openhab.binding.freeathomesystem/src/main/resources/OH-INF/i18n/freeathomesystem.properties

```diff
@@ -0,0 +1,652 @@
+# add-on
+
+addon.freeathomesystem.name = FreeAtHomeSystem Binding
+addon.freeathomesystem.description = This is the binding for free@home system.
+
+# thing types
+
+thing-type.freeathomesystem.device.label = free@home Device
+thing-type.freeathomesystem.device.description = This represents the generic free@home device
+thing-type.freeathomesystem.gateway.label = Free@Home Gateway
```

Comment:
It looks like this file needs to be [regenerated](https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file) as this label was changed in the thing type definition.

Reply:
**Fixed**